lib/utils.js

```diff
@@ -205,6 +205,27 @@ utils.getResourceUID = function getResourceUID(namespace, resource) {
         .update(namespace + resource, 'binary').digest('hex');
 };
 
+/**
+ * Validate request headers to determine if readable characters
+ * @param {object} headers - headers attached to the http request (lowercased)
+ * @return {boolean} - returns true if headers are readable, false if not
+ */
+utils.checkReadableHeaders = function checkReadableHeaders(headers) {
```

Comment:
@MathieuCassagne - Yep, `every` method return true if the callback function returns a truthy value for every array element; otherwise, false.